src/main/java/uk/gov/hmcts/reform/notificationservice/service/NotificationService.java

```diff
@@ -70,33 +72,26 @@ private void processNotifications(Notification notification) {
             notificationRepository.markAsSent(notification.id, response.getNotificationId());
 
             log.info(
-                "Error notification sent. Service: {}, Zip file: {}, ID: {}, Notification ID: {}",
-                notification.service,
-                notification.zipFileName,
-                notification.id,
+                "Error notification sent. {}. Notification ID: {}",
+                notification.basicInfo(),
                 response.getNotificationId()
             );
-        } catch (FeignException.BadRequest | FeignException.UnprocessableEntity exception) {
+        } catch (BadRequest | UnprocessableEntity exception) {
             log.error(
-                "Received http status {} from client. Marking as failure. "
-                    + "Service: {}, Zip file: {}, ID: {}, Client response: {}",
+                "Received http status {} from client. Marking as failure. {}. Client response: {}",
                 exception.status(),
-                notification.service,
-                notification.zipFileName,
-                notification.id,
+                notification.basicInfo(),
                 exception.contentUTF8(),
```

Comment:
Not strictly this PR, but while we're improving the logging, might be worth making contentUTF8() safe (it will throw exception when content is null).

Reply:
doesn't look like it will throw an exception:

```java
public String contentUTF8() {
    if (responseBody != null) {
        return new String(responseBody, UTF_8);
    } else {
        return "";
    }
}
```